test/units/parsing/test_ajson.py

```diff
@@ -20,3 +25,159 @@ def test_AnsibleJSONDecoder_vault():
     assert isinstance(data['password'], AnsibleVaultEncryptedUnicode)
     assert isinstance(data['bar']['baz'][0]['password'], AnsibleVaultEncryptedUnicode)
     assert isinstance(data['foo']['password'], AnsibleVaultEncryptedUnicode)
+
+
+class TestAnsibleJSONEncoder():
+    """
+    Class for testing AnsibleJSONEncoder
+    """
+
+    #################
+    # Fixture section
+
+    @pytest.fixture
+    def mapping(self, request):
```

Comment:
Include a comment about why you're using this class instead of a plain dict.

I assume you chose this approach so that the interface exposed is Mapping instead of MutableMapping.

Reply:
Done, thank you